docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -11,27 +11,21 @@
 Lifecycle policies allow you to automate the lifecycle of your APM indices as they grow and age.
 A default policy is applied to each APM data stream, but can be customized depending on your business needs.
 
-[discrete]
-[id="index-lifecycle-policies-default{append-legacy}"]
-== Default policies
+In the latest version of Elastic APM, clusters are managed by
+{ref}/index-lifecycle-management.html[index lifecycle management (ILM)]
+to provide the default data retention settings for APM data as well as allow
+customization for lifecycle.
 
 [NOTE]
 ====
-// Explain what changed
-// Before
-Before 8.15, clusters used {ref}/index-lifecycle-management.html[index lifecycle management (ILM)] to provide the default data retention settings for APM data.
-// After
-As of 8.15, new clusters use {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide default data retention settings, but the customization for lifecycle is still performed using ILM policies, which are now configured to override the default DSL polices.
-
-// What this means for existing clusters that are upgraded
-For _existing_ clusters upgrading to 8.15, the default ILM policies should still work as expected.
-You can can continue using ILM or switch the default to DSL explicitly using the {ref}/data-stream-apis.html[data stream API].
-
-// What this means for new clusters that are created
-For _new_ clusters created in 8.15 or later, if you prefer to continue using ILM,
-follow the steps in this guide to create a custom ILM policy and add it to the `*@custom` component template for each data stream.
+Some older indices might be managed by {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] if cluster is upgraded to 8.17.
+More details are available in <<apm-release-notes>>.
```

Comment:
Are there going to be more details in the 8.17 release notes? It would be nice to link to something more specific than the general release notes page. I'm kind of surprised there's no release notes page for 8.17 on the `main` / `master` branch. 🤔

Reply:
@1pkg can you ensure the release notes are on `main`, `8.x` and `8.17` branch please?

Reply:
@colleenmcginnis Given that we will have the release notes soonish, should we go ahead with merging this first?

Reply:
@simitt here's the PR https://github.com/elastic/apm-server/pull/14929/files where I manually copied 8.17 changelog to the main branch. I'm not sure why, but the current minor release automation only creates changelog for specific 8.x and 8.17+ branches.

Reply:
The PR with the changelog from above was merged.

Reply:
Thanks @1pkg! @lahsivjar I think we should update the release note link to the 8.17 page so it's easier for users to find the relevant information. What do you think?

```suggestion
More details are available in <<apm-release-notes-8.17>>.
```

Reply:
I think we need this PR, too: https://github.com/elastic/apm-server/pull/14932.

Reply:
Thanks @colleenmcginnis for raising this PR

Reply:
@lahsivjar I'll rebuild this PR after https://github.com/elastic/apm-server/pull/14932 is merged and set this PR to automerge. 😅

Reply:
Thank you!